pages/interface/components/ReadTime/index.js

```diff
@@ -0,0 +1,14 @@
+import { Text } from '@/TabNewsUI';
+
+function getReadTime(text, wpm = 260) {
```

Comment:
Fiquei um pouco na dúvida de em qual unidade de medida iria retornar, talvez explicitar isto facilite a leitura, o que acha?

```suggestion
function getReadTimeInMinutes(text, wpm = 260) {
```

Reply:
@RobertDS07 boa, feito!